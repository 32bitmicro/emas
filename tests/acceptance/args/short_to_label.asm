label1:
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
label2:
nop nop
ujs label1
ujs label2
ujs label3
ujs label4
nop nop nop nop nop nop nop nop nop nop
label3:
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop
label4:
