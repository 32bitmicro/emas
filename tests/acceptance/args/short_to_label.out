@ 0x0000 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0001 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0002 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0003 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0004 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0005 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0006 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0007 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0008 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0009 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000e : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000f : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0010 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0011 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0012 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0013 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0014 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0015 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0016 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0017 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0018 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0019 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x001a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x001b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x001c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x001d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x001e : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x001f : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0020 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0021 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0022 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0023 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0024 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0025 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0026 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0027 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0028 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0029 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x002a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x002b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x002c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x002d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x002e : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x002f : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0030 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0031 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0032 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0033 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0034 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0035 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0036 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0037 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0038 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0039 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x003a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x003b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x003c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x003d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x003e : 0xe23f  /  111 000 1 000 111 111  /  57919
@ 0x003f : 0xe204  /  111 000 1 000 000 100  /  57860
@ 0x0040 : 0xe00b  /  111 000 0 000 001 011  /  57355
@ 0x0041 : 0xe03f  /  111 000 0 000 111 111  /  57407
@ 0x0042 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0043 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0044 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0045 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0046 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0047 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0048 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0049 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x004a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x004b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x004c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x004d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x004e : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x004f : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0050 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0051 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0052 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0053 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0054 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0055 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0056 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0057 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0058 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0059 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x005a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x005b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x005c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x005d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x005e : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x005f : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0060 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0061 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0062 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0063 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0064 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0065 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0066 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0067 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0068 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0069 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x006a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x006b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x006c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x006d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x006e : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x006f : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0070 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0071 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0072 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0073 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0074 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0075 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0076 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0077 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0078 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0079 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x007a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x007b : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x007c : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x007d : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x007e : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x007f : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0080 : 0xe000  /  111 000 0 000 000 000  /  57344
